Enterprise-CMCS/macpro-mdct-qmr
# HPC-AD Update To HBD-AD

### Description

For Reporting Year 2023, previous Adult Measure **HPC-AD** (Comprehensive Diabetes Care: Hemoglobin A1c (HbA1c) Poor Control (> 9.0%)) is now being renamed to **HBD-AD** (Hemoglobin A1c Control for Patients with Diabetes)

Changes made to update the abbreviation: 
- Title change
- New rate labels were added
- New questionNote variable added to display text after list
- Update measure description
- New HBD-AD cypress Test

### Related ticket(s)


MDCT-2351
MDCT-2359 (Only HBD-AD N/D/R, Page 2)

---
### How to test

1) Go to Reporting Year "2023 Core Set"
2) Select Adult Core Set Measures
3) There should now be an option for HBD-AD and no more HPC-AD
4) See MDCT-2351 for description of measure text changes
5) See MDCT-2359 (PDF Page 2) for N/D/R text changes 

### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
